### ISSUE TITLE
common identifier name for node ids

### ISSUE THREAD

**Issue by jbaron** [OpenZWave/node-openzwave-shared#63]
Been using this library for a few weeks now and it is working nicely, so thanks a lot. Not a single crash so far my dev machine (although to be fair I am only using basic features).

One thing I noticed is that when I get a value the node identifier is called node_id. However when I want to set a value I have to use nodeid (so without the underscore). This makes reuse of the previous received data just that bit harder. 

I know it can be fixed easily in "getZwaveValueID", but I was just wondering if there is a reason for this? 

P.S Since I'm using it in a TypeScript environment, I also start creating a declaration file so it I have some type checking in place.  Will share this file once in a decent shape.

**Comment by ekarak** [comment]
you're right, I'll fix this inconsistency.

**Comment by ekarak** [comment]
Hello all,
I've just pushed [a fix in a new bugfix branch](https://github.com/OpenZWave/node-openzwave-shared/tree/bugfix/issue-63-getZwaveValueID-typo). Can you all please download this branch and test it? 
Apologies again, this all was due to a silly typo (I was checking for 'nodeid' object property, not the correct 'node_id').

Thanks!

**Comment by jbaron** [comment]
That is a really quick fix, thanks a lot!!!

**Comment by jbaron** [comment]
Compiled it and seems to be working fine. I did however notice a difference at startup (scanning phase). My previous version of open-zwave-shared showed also all the configuration commands as "value add". 

Now however I only see values of the active sensor. Has somehow this behaviour changed (I think my previous version was already month old)?

**Comment by ekarak** [comment]
The node.js plugin does not filter or alter the events raised by the
underlying c++ library. Have you updated your Openzwave sources perhaps?
Στις 7 Φεβ 2016 12:57 μ.μ., ο χρήστης "Peter" <notifications@github.com>
έγραψε:
>
> Compiled it and seems to be working fine. I did however notice a
difference at startup (scanning phase). My previous version of
open-zwave-shared showed also all the configuration commands as "value add".
>
> Now however I only see values of the active sensor. Has somehow this
behaviour changed (I think my previous version was already month old)?

**Comment by jbaron** [comment]
Just figured out the underlying issue (open-zwave still has still many mysteries to me ;). 

The zwave config file (the xml one) from my previous version has all the command classes in there because when I added my device it broadcasted all its command classes at that time and open-wave stored that info in the XML file. This new node-wave-shared version does’t know of that and created a fresh also empty XML file. So I copied over the old config file and everything is fine again.


— Peter
